tests/config/databend-query-node-1.toml

```diff
@@ -60,46 +59,46 @@ auth_string = "3081f32caef285c232d066033c89a78d88a6d8a5"
 name = "ping"
 definition = "CREATE FUNCTION ping(STRING) RETURNS STRING LANGUAGE python HANDLER = 'ping' ADDRESS = 'http://0.0.0.0:8815'"
 
+[query.settings]
+aggregate_spilling_memory_ratio = 60
+join_spilling_memory_ratio = 60
+
 [log]
+level = "INFO"
 
 [log.file]
-level = "INFO"
 format = "text"
-dir = "./.databend/logs_3"
-
-[log.structlog]
-on = true
-dir = "./.databend/structlog_3"
+dir = "./.databend/logs_1"
 
 [meta]
 # It is a list of `grpc_api_advertise_host:<grpc-api-port>` of databend-meta config
-endpoints = ["0.0.0.0:9191"]
+endpoints = ["meta:9191"]
 username = "root"
 password = "root"
 client_timeout_in_second = 60
 auto_sync_interval = 60
 
 # Storage config.
 [storage]
-# fs | s3 | azblob | obs | oss
-type = "fs"
+type = "s3"
 
-# Set a local folder to store your data.
-# Comment out this block if you're NOT using local file system as storage.
-[storage.fs]
-data_path = "./.databend/stateless_test_data"
+[storage.s3]
+bucket = "databend"
+endpoint_url = "http://minio:9000"
+#endpoint_url = "http://localhost:9000"
+access_key_id = "minioadmin"
+secret_access_key = "minioadmin"
+enable_virtual_host_style = false
 
 # Cache config.
 [cache]
-# Type of storage to keep the table data cache
-#
-# available options: [none|disk]
-# default is "none", which disable table data cache
-# use "disk" to enabled disk cache
 data_cache_storage = "none"
 
 [cache.disk]
 # cache path
 path = "./.databend/_cache"
 # max bytes of cached data 20G
 max_bytes = 21474836480
+
+[spill]
+spill_local_disk_path = "./.databend/temp/_query_spill"
```

Comment:
I found that the `spill_local_disk_path` will be deprecated in newer databend-query according to https://docs.databend.com/guides/deploy/references/node-config/query-config#spill-section

Reply:
updated